app/templates/identity.hbs

```diff
@@ -1,7 +1,7 @@
 {{page-title 'Identity'}}
 
-<h1 class="identity-heading">
-    Identity Service Verification
+<h1 class='identity-heading'>
+  Identity Service Verification
```

Comment:
Can we please fix/revert these quote changes?

Reply:
fixed the quote changes